packages/razzle/package.json

```diff
@@ -29,44 +29,47 @@
     "reasonml"
   ],
   "dependencies": {
-    "assets-webpack-plugin": "3.5.1",
-    "autoprefixer": "7.1.1",
-    "babel-core": "6.25.0",
-    "babel-eslint": "^8.2.3",
-    "babel-jest": "20.0.3",
-    "babel-loader": "7.1.4",
-    "babel-preset-razzle": "^2.0.0",
-    "chalk": "1.1.3",
-    "css-loader": "0.28.11",
-    "dotenv": "4.0.0",
-    "eslint": "4.19.1",
-    "eslint-config-react-app": "^2.1.0",
-    "eslint-loader": "^2.0.0",
-    "eslint-plugin-flowtype": "^2.35.0",
-    "eslint-plugin-import": "^2.7.0",
-    "eslint-plugin-jsx-a11y": "^5.1.1",
-    "eslint-plugin-react": "^7.3.0",
-    "file-loader": "^1.1.11",
-    "fs-extra": "3.0.1",
-    "jest": "^23.1.0",
-    "mini-css-extract-plugin": "^0.4.0",
+    "@babel/core": "7.1.2",
+    "assets-webpack-plugin": "3.9.7",
+    "babel-core": "^7.0.0-bridge.0",
```

Comment:
@jaredpalmer is `babel-core` required if have `@babel/core` already?

Reply:
https://github.com/babel/babel-bridge